src/envshortcut.cpp

```diff
@@ -147,7 +148,7 @@ Shortcut::Shortcut(const Executable& exe)
 {
   const auto i = *InstanceManager::singleton().currentInstance();
 
-  m_name = exe.title();
+  m_name = MOBase::sanitizeFileName(exe.title());
```

Comment:
What's that?

Reply:
I have no clue... It doesn't show up when I do a local `git show`, so I don't know why it's included in that commit...